packages/react-theme-provider/etc/react-theme-provider.api.md

```diff
@@ -0,0 +1,43 @@
+## API Report File for "@fluentui/react-theme-provider"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as React from 'react';
+
+// Warning: (ae-forgotten-export) The symbol "RecursivePartial" needs to be exported by the entry point index.d.ts
+//
+// @public (undocumented)
+export interface Theme extends RecursivePartial<ThemePrepared> {
+}
+
+// @public (undocumented)
+export interface ThemePrepared {
+    // (undocumented)
+    stylesheets: string[];
+    // (undocumented)
+    tokens: {
+        site: {
+            body: ThemePlateSet;
+            accent: ThemePlateSet;
+            neutral: ThemePlateSet;
+        };
+        [key: string]: TokenSetType;
+    };
+}
+
+// Warning: (ae-forgotten-export) The symbol "ThemeProviderProps" needs to be exported by the entry point index.d.ts

```

Comment:
Should probably fix this and the similar warnings by exporting all appropriate things

Reply:
Fixed up some of the typings and made sure they're all exported.